packages/react-draggable-dialog/src/components/DraggableDialogHandle/DraggableDialogHandle.test.tsx

```diff
@@ -7,19 +7,15 @@ import * as contexts from '../../contexts/DraggableDialogContext';
 import * as dnd from '@dnd-kit/core';
 import { DraggableDialogHandleState } from './DraggableDialogHandle.types';
 
-jest.mock('../../contexts/DraggableDialogContext', () => {
-  return {
-    __esModule: true,
-    ...jest.requireActual('../../contexts/DraggableDialogContext'),
-  };
-});
-
-jest.mock('@dnd-kit/core', () => {
-  return {
-    __esModule: true,
-    ...jest.requireActual('@dnd-kit/core'),
-  };
-});
+jest.mock('../../contexts/DraggableDialogContext', () => ({
+  __esModule: true,
+  ...jest.requireActual('../../contexts/DraggableDialogContext'),
```

Comment:
why is this needed ? mocking whole module and using real implementation ?

Reply:
This is needed to be able to Spy on functions that come from that module. Whatever Jest does, hoisting the variables when compiling the test files, doesn't work with ES Modules.

Reply:
if we really sure we need such amount of mocks etc, we should be explicit.

Take a look at this https://github.com/microsoft/fluentui-contrib/pull/129/commits/c68318d055eee5b00fa2ac9c46ffff99668d3d68

we can talk about it more offline

Reply:
This was added before the availability of the E2E tests. I'm cancelling this PR and I'll those tests in a separate PR in favor of the E2E tests.